webapp/graphite/render/evaluator.py

```diff
@@ -29,18 +34,19 @@ def evaluateTarget(requestContext, targets, noPrefetch=False):
 
   return seriesList
 
+
 def evaluateTokens(requestContext, tokens, replacements=None, pipedArg=None):
   if tokens.template:
     arglist = dict()
     if tokens.template.kwargs:
-      arglist.update(dict([(kwarg.argname, evaluateTokens(requestContext, kwarg.args[0])) for kwarg in tokens.template.kwargs]))
+      arglist.update(dict([(kwarg.argname, evaluateScalarTokens(kwarg.args[0])) for kwarg in tokens.template.kwargs]))
     if tokens.template.args:
-      arglist.update(dict([(str(i+1), evaluateTokens(requestContext, arg)) for i, arg in enumerate(tokens.template.args)]))
+      arglist.update(dict([(str(i+1), evaluateScalarTokens(arg)) for i, arg in enumerate(tokens.template.args)]))
```

Comment:
This change ensures that template arg values are scalars, to be consistent with documentation and match a bugfix in `extractPathExpressions` which didn't handle templates properly.